pkg/ipcache/metadata_test.go

```diff
@@ -782,18 +783,25 @@ func TestMetadataRevision(t *testing.T) {
 func TestMetadataWaitForRevision(t *testing.T) {
 	m := newMetadata()
 
-	p1 := netip.MustParsePrefix("1.1.1.1/32")
-	wantRev := m.enqueuePrefixUpdates(p1)
+	_, wantRev := m.dequeuePrefixUpdates()
```

Comment:
What is the first value that we ignore? 
When I check the signature there are only the modifiedPrefixes which are returned.

Reply:
Yes - but we haven't modified any prefixes, so we know this is empty and thus can ignore it. We're basically only using `dequeuePrefixUpdates`  to obtain the next revision, as `dequeuePrefixUpdates` always bumps the revision if called - even if there's nothing queued

Reply:
I was checking the signature of the function on a older branch, my bad!